lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
 
+    //Const for main stylesheet.
     const stylesheet = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
-    stylesheet.rel = 'stylesheet';
     stylesheet.href = 'chrome://universalsearch-skin/content/style.css';
+    stylesheet.rel = 'stylesheet';
     stylesheet.type = 'text/css';
     stylesheet.id = 'universal-search-stylesheet';
     stylesheet.style.display = 'none';
 
+    //Const for platform-specific stylesheet.
+    const stylesheetPlatform = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
+    stylesheetPlatform.rel = 'stylesheet';
+    stylesheetPlatform.type = 'text/css';
+    stylesheetPlatform.id = 'universal-search-stylesheetPlatform';
```

Comment:
addresses a few of the above line notes
<img width="634" alt="screen shot 2016-03-21 at 8 40 57 pm" src="https://cloud.githubusercontent.com/assets/9204835/13938732/ac4c7a2a-efa4-11e5-856b-bb3aee1fb429.png">